rediscluster/client.py

```diff
@@ -410,11 +410,13 @@ def execute_command(self, *args, **kwargs):
                 return self.parse_response(r, command, **kwargs)
             except (RedisClusterException, BusyLoadingError):
                 raise
-            except (ConnectionError, TimeoutError):
-                try_random_node = True
-
+            except ConnectionError:
+                r.disconnect()
+            except TimeoutError:
                 if ttl < self.RedisClusterRequestTTL / 2:
-                    time.sleep(0.1)
+                    time.sleep(0.05)
```

Comment:
NOTE: lowered this value so that this is handled the same as:
```
            except TryAgainError as e:
                if ttl < self.RedisClusterRequestTTL / 2:
                    time.sleep(0.05)
```
below.  If there's a reason these should be different, let me know and I'll switch it back.